### DISCUSSION TITLE
Feature Request: User-Defined Tag/Author characters

### DISCUSSION THREAD

**Discussion by nonplayer** [damionx7/Zettel-Notes-Documentation#156] [Feature Requests]
**Request:** It would be awesome if there was a user-side option to set a custom tag and author identifying character

**Use Case:** My use case here is two-fold.

First, I use my markdown notes repo to store, among many other things, preformatted social media post templates for quick copy-paste posting. Most of these templates include one or more hash-tags, for use with those social media platforms. Since Zettel notes by default reads those hashtags as note tags, there are so many of them that the Zettel Notes tags section is now made completely useless to me, as it is flooded with these that I don't consider note tags. Being able to specify the prefix character that it identifies as a note tag would eliminate this issue.

Second, I sync my notes repo across Windows, Mac, Linux, Android, and iOS as I make use of all of these devices in my personal life and work. With the desktops specifically, I use the app QOwnNotes as my main source of truth. That app has the ability to read @-prefixed tags embedded inside notes to generate and catalog tags (via a script extension). When synched to Android, Zettel Notes incorrectly reads all of these as authors, but I'd prefer they were read as tags. This would also be eliminated by the ability to specify the tag prefix character.

Thanks for your time and consideration!

**Comment by damionx7** [comment]
Hey. I have added a new feature. User can define custom attribute like tags and authors and its secret words. So, you can have a new attribute called TAG-2 with secret word as @ . Clicking on this custom attribute will list all its occurences in the notes just like tags and authors.

**Reply by nonplayer** [reply]
@damionx7 thanks for adding this! ONe question now, I'm a bit confused at one aspect of the new feature. When I go to add a new attribute, it asks for a Name, a drop-down symbol, and then a "Yaml code" - what is that last part supposed to be?

**Reply by damionx7** [reply]
Hey. eg. or YAML code - category

```
---
category: video
---
```

You use this as yaml frontmatter in your notes.

**Reply by nonplayer** [reply]
Thanks for the clarification! I am encountering some issues/requests with this which I'll post below.

First, seems not all the symbols work. For instance, if I try to set to the carat ^ symbol, it crashes the app.

Second, it'd be neat if there were a way to have it set to an AND/OR option with the Symbol and the Yaml Code, so that the user can choose to only show items with a YAML code, and ignore the symbols.

Third, I imagine this might not be possible, but if the tool could be set to use double characters - such as "@@tag" in format, that could make it easier to further distinguish custom tags.

Thanks again for your responses!

**Reply by damionx7** [reply]
I will try to fix these issues.

**Reply by nonplayer** [reply]
Thank you!
